paymentsheet/src/test/java/com/stripe/android/paymentsheet/ui/PaymentOptionScreenshotTest.kt

```diff
@@ -99,6 +102,29 @@ class PaymentOptionScreenshotTest {
         }
     }
 
+    @Test
+    fun testConfirmRemoveDialog() {
```

Comment:
Not sure if this is necessary.

Reply:
I will keep it, I think it is nice to see how the dialog changes with the appearance API.